saleor/discount/migrations/0047_move_codes_to_new_model.py

```diff
@@ -0,0 +1,56 @@
+# Generated by Django 3.2.20 on 2023-08-21 07:23
+
+from django.db import migrations
+
+
+BATCH_SIZE = 5000
+
+
+def queryset_in_batches(queryset):
+    start_pk = 0
+
+    while True:
+        qs = queryset.order_by("pk").filter(pk__gt=start_pk)[:BATCH_SIZE]
+        pks = list(qs.values_list("pk", flat=True))
+
+        if not pks:
+            break
+
+        yield pks
+
+        start_pk = pks[-1]
+
+
+def move_codes_to_new_model(apps, schema_editor):
+    Voucher = apps.get_model("discount", "Voucher")
+    VoucherCode = apps.get_model("discount", "VoucherCode")
+
+    voucher_codes = []
```

Comment:
Did you test the solution on big DB? 
`voucher_codes = []` should be moved inside batch, otherwise we will try to recreate already created vouchers and the error will be raised